home-manager/home-manager

```diff
@@ -263,8 +264,9 @@ function doBuild() {
     newsInfo=$(buildNews)
 
     local exitCode
-
-    doBuildAttr --attr activationPackage \
+    doBuildAttr \
+                ${NO_OUT_LINK+--no-out-link} \
+                --attr activationPackage \
```

Comment:
```suggestion
    doBuildAttr ${NO_OUT_LINK+--no-out-link} \
                --attr activationPackage \
```

Reply:
I did it this way to match the other calls to `doBuildAttr`. If the style should change then better do all of them together in one commit.